src/Euclidean.jl

```diff
@@ -9,20 +9,38 @@ Euclidean vector space $\mathbb R^n$.
 
 generates the $n$-dimensional vector space $\mathbb R^n$.
 
-   Euclidean(m, n)
+   Euclidean(n₁,n₂,...)
 
-generates the $mn$-dimensional vector space $\mathbb R^{m \times n}$, whose
-elements are interpreted as $m \times n$ matrices.
+generates the $mn$-dimensional vector space $\mathbb R^{n_1\times n_2\times \cdots}$, whose
```

Comment:
I'd specify the number of indices, for example `n_1, n_2, \dots, n_k` (to make it clear that there are finitely many indices). And it's no longer true that it's `mn`-dimensional.

Reply:
i chose subscript i, but yes that's a good change, such that the reader can understand that better.